src/ansys/mapdl/core/launcher.py

```diff
@@ -309,7 +308,7 @@ def generate_mapdl_launch_command(
     ram: Optional[int] = None,
     port: int = MAPDL_DEFAULT_PORT,
     additional_switches: str = "",
-) -> str:
+) -> List[str]:
```

Comment:
```suggestion
) -> list[str]:
```
Why not use the Python 3.10+ typehint version?